ntbs-service/DataMigration/NotificationImportService.cs

```diff
@@ -111,6 +108,16 @@ private async Task<ImportResult> ValidateAndImportNotificationGroupAsync(Perform
             LookupAndAssignPostcode(notifications);
 
             _logger.LogInformation(context, requestId, $"{notifications.Count} notifications found to import for {patientName}");
+            
+            // Verify that no repeated NotificationIds have returned

```

Comment:
just out of interest, how often does this happen?

Reply:
Not often at all, but i've seen an example where there are two addresses for a notification, so the left join creates two cases